modules/programs/zathura.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ...}:
+
+with lib;
+
+let
+
+  cfg = config.programs.zathura;
+
+  formatLine = n: v:
+    let
+      formatValue = v:
+        if isBool v then (if v then "true" else "false")
+        else toString v;
+    in
+      "set ${n}\t\"${formatValue v}\"";
+in
+
+{
+  meta.maintainers = [maintainers.rprospero];
+
+  options.programs.zathura = {
+    enable = mkEnableOption ''Zathura is a highly customizable and funtional
+      document viewer focused on keyboard interaction.'';
+    options = mkOption {
+      default = null;
+      type = types.nullOr types.attrs;
+      description = ''
+      Add :set command options to zathura and make them permanent.
+      Run <code>man zathura</code> to see the full list of options
+      '';
+      example = literalExample ''
+        {default-bg = "#000000"; default-fg = "#FFFFFF";}
```

Comment:
Since this is on one line anyway you could just do
```nix
example = { default-bg = "#000000"; default-fg = "#FFFFFF"; };
```

Reply:
That was an unnecessary indirection.